SPECS/qt5-qtbase/qt5-qtbase.spec

```diff
@@ -61,8 +61,11 @@ BuildRequires: zlib-devel
 BuildRequires: qt5-rpm-macros
 
 Requires:         icu
+Requires:         pcre2
+Requires:         dbus
 Requires(post):   chkconfig
 Requires(postun): chkconfig
+AutoReq:        no
```

Comment:
Similar question about `AutoReq` as in `calamares.spec`.

Reply:
Provided update above is applicable here also.